repotracker/repotracker.go

```diff
@@ -302,6 +292,16 @@ func (repoTracker *RepoTracker) StoreRevisions(ctx context.Context, revisions []
 				return err
 			}
 		}
+		if pInfo.Project == nil {
+			msg := fmt.Sprintf("unable to find project config for revision %s", revision)
```

Comment:
Why do we need msg here? Wouldn't we return err instead of errors.New(msg)?

Reply:
I think Project being nil logging is also probably handled by the err != nil case above so we can probably just return.